app/src/androidTest/java/com/github/fribourgsdp/radio/activities/GameSettingsActivityTest.kt

```diff
@@ -67,6 +87,15 @@ class GameSettingsActivityTest {
         val withHint = true
         val private = true
 
+        onView(withId(R.id.chooseTimeButton))
+            .perform(ViewActions.click())
+        onView(withText("OK"))
+            .check(matches(isDisplayed()))
+        onView(withText("OK"))
+            .perform(ViewActions.click())
```

Comment:
I don't know if it changes anything runtime wise but you can merge this two:
```kotlin
onView(withText("OK"))
    .check(matches(isDisplayed()))
    .perform(ViewActions.click())
```

Reply:
Done.